machine/machine_core/machine.py

```diff
@@ -230,6 +230,10 @@ def allowed_messages(self):
             # but we have to ignore that general header line
             allowed.append('^Stack trace of.*')
 
+        if self.image in ['rhel-8-5']:
```

Comment:
Needs "distropkg" as well: [log](https://logs.cockpit-project.org/logs/pull-2085-20210610-053945-815f67ea-rhel-8-5-distropkg-cockpit-project-cockpit/log.html#90)

Reply:
And for some reason [does not work for c-machines](https://logs.cockpit-project.org/logs/pull-2085-20210610-053939-815f67ea-rhel-8-5-cockpit-project-cockpit-machinesn/log.html#5)

Reply:
Fixed for distropkg, sent https://github.com/cockpit-project/cockpit-machines/pull/201 for c-machines, tracking in todo list now.